common-develocity-sbt-configuration/build.sbt

```
@@ -1,13 +1,12 @@
 // You can configure the Develocity sbt plugin to publish Build Scans to your Develocity server
 // by adding the following configuration to your project's build.sbt file
 
-Global / gradleEnterpriseConfiguration :=
-  GradleEnterpriseConfiguration(
+Global / develocityConfiguration :=
```

```suggestion
ThisBuild / develocityConfiguration :=
```
We recommend scoping the settings in `ThisBuild` now